napari/layers/base/base.py

```diff
@@ -20,6 +20,85 @@ class Layer(VisualWrapper, KeymapMixin, ABC):
+        alpha values of the layer visual get mixed. Allowed values are
+        {'opaque', 'translucent', and 'additive'}.
+    visible : bool
+        Whether the layer visual is currently being displayed.
+
+
+    Attributes
+    ----------
+    name : str
+        Unique name of the layer.
+    opacity : flaot
+        Opacity of the layer visual, between 0.0 and 1.0.
+    visible : bool
+        Whether the layer visual is currently being displayed.
+    blending : Blending
+        Determines how RGB and alpha values get mixed.
+            Blending.OPAQUE
+                Allows for only the top layer to be visible and corresponds to
+                depth_test=True, cull_face=False, blend=False.
+            Blending.TRANSLUCENT
+                Allows for multiple layers to be blended with different opacity
+                and corresponds to depth_test=True, cull_face=False,
+                blend=True, blend_func=('src_alpha', 'one_minus_src_alpha').
+            Blending.ADDITIVE
+                Allows for multiple layers to be blended together with
+                different colors and opacity. Useful for creating overlays. It
+                corresponds to depth_test=False, cull_face=False, blend=True,
+                blend_func=('src_alpha', 'one').
+    scale : sequence of float
+        Scale factors for the layer visual in the scenecanvas.
+    translate : sequence of float
+        Translation values for the layer visual in the scenecanvas.
+    z_index : int
+        Depth of the layer visual relative to other visuals in the scenecanvas.
+    coordinates : tuple of float
+        Coordinates of the cursor in the image space of each layer. The length
+        of the tuple is equal to the number of dimensions of the layer.
+    indices : tuple of int or Slice
+        Used for slicing arrays on each dimension.
+    position : 2-tuple of int
+        Cursor position in canvas ordered (x, y).
+    shape : tuple of int
+        Size of the data in the layer.
+    range : list of 3-tuple of int
+        Ranges of data for slicing specifed by (min, max, step), one for each
+        axis.
+    ndim : int
+        Dimensionality of the layer.
+    selected : bool
+        Flag if layer is selected in the viewer or not.
+    thumbnail : (N, M, 4) array
+        Array of thumbnail data for the layer.
+    status : str
+        Displayed in status bar bottom left.
+    help : str
+        Displayed in status bar bottom right.
+    interactive : bool
+        Determine if canvas pan/zoom interactivity is enabled.
+    cursor : str
+        String identifying which cursor displayed over canvas.
+    cursor_size : int | None
+        Size of cursor if custom. None yields default size
+    scale_factor : float
+        Conversion factor from canvas coordinates to image coordinates, which
+        depends on the current zoom level.
+
+    Extended Summary
+    ----------
```

Comment:
"underline" should match length of corresponding header

Reply:
fixed, but we'll need to fix elsewhere too